### ISSUE TITLE
[future] django will change the behaviour how to handle/re-render rendered template

### ISSUE THREAD

**Issue by chenull** [viewflow/django-material#222]
this is currently only applies to django master

the frontend menu of `django-material` is defined in this [code](https://github.com/viewflow/django-material/blob/master/material/frontend/apps.py#L104-L118). Using django master (1.11-alpha), rendering menu will raise:
```
  File "/Users/ayik/Repo/Django/django/django/template/context.py", line 285, in make_context
    raise TypeError('context must be a dict rather than %s.' % context.__class__.__name__)
TypeError: context must be a dict rather than RequestContext.
[11/Jan/2017 16:16:29] "GET /employees/departments/ HTTP/1.1" 500 321465
```

i submitted a [bug](https://code.djangoproject.com/ticket/27722) in django trac, but @timgraham said that it was intended new behaviour according to [this discussion](https://code.djangoproject.com/ticket/27258)

**Comment by kmmbvnr** [comment]
Thanks for the report. There are always a lot of incompatible changes over django releases that affects such low-level dependant libs like django-material. For sure I'll work on this, but only after django 1.11 release.

**Comment by auvipy** [comment]
alpha1 is released

**Comment by kmmbvnr** [comment]
I made a new ticket - https://code.djangoproject.com/ticket/27770#ticket

This is clear new django version mistake, that's usually became fixed till the X.X.1 release.

That's why I usually prefer, not even take a look into even to alpha versions.

**Comment by timgraham** [comment]
From a Django maintainer perspective, I appreciate all the testing we can get with prelease versions so we can fix issues before the final release.

**Comment by auvipy** [comment]
@kmmbvnr drf also added alpha versions testing https://github.com/tomchristie/django-rest-framework/commit/5e9249f42926a238dc03e70f7605a9417de0fc44

**Comment by kmmbvnr** [comment]
Sorry, I'm not in this race.
